packages/react-next/src/components/Checkbox/__snapshots__/Checkbox.test.tsx.snap

```diff
@@ -217,10 +215,8 @@ exports[`Checkbox renders indeterminate correctly 1`] = `
         @media screen and (-ms-high-contrast: active){.ms-Fabric--isFocusVisible &:focus + label::before {
           outline: 1px solid ActiveBorder;
         }
-    id="checkbox-2"
-    onBlur={[Function]}
```

Comment:
how come Checkbox snapshot has changes?

Reply:
beforeEach resetIds. We should not allow ids to make their way into snapshots, creates fragility.

Reply:
but why is `onFocus` and `onBlur` removed?

Reply:
The old Checkbox would always define onFocus and onBlur.

In the function component refactor, these 2 are only set if the user passes in inputProps. So the change is expected.

We don't know how this passed in Michael's refactor though, so maybe a bag merge was involved.